contribs/gmf/less/lidarprofile.less

```diff
@@ -0,0 +1,58 @@
+@lidar-profile-height: 35rem;
+.gmf-lidar-profile-container {
+  position: relative;
+  overflow: hidden;
+  @profile-legend-width: 15rem;
```

Comment:
Strange notation here, what that means ?

Reply:
rem: Relative to font-size of the root element. Used already in raster profile. More info [here](https://www.w3schools.com/cssref/css_units.asp)

Reply:
That's not the problem, what I didn't understand was the @profile-legend-width declared here. Now I understand a little bit but why here ? That looks strange

Reply:
Ok, @profile-legend-width moved to more logical position in less file

Reply:
Fixed, moved higher in less file